### PR TITLE
alert buttons done

### PR DESCRIPTION
**Pull request by kushalsourav**
alert and buttons finished

### CODE REVIEW

**Comment by mayuresh-tech** [comment]
Code is clean and UI is responsive!

Bugs to fix (suggestions):
Fix line-height in text badge.
Fix code section in mobile screens.

File: packages/components/alert.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
+.alert {
+    font-size: 0.875rem;
+    font-weight: 600;
+    padding: 1rem;
+    line-height: 1.25rem;
+    letter-spacing: 1px;
+    text-transform: capitalize;
+    width: 100%;
+    height: 3rem;
+    border-radius: 0.188rem;
```
Comment (nisargpawade) [inline packages/components/alert.css]: you can use 1-10px if units are under 1rem

File: packages/components/alert.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
+.alert {
+    font-size: 0.875rem;
+    font-weight: 600;
+    padding: 1rem;
+    line-height: 1.25rem;
```
Comment (partha8) [inline packages/components/alert.css]: try having line height in unitless terms
like 1.6 or something

File: packages/components/alert.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    color: #0000cd;
+    border: none;
+    text-transform: uppercase;
+}
+
+.btn-text:focus {
+    padding: 0.625rem 1rem;
+    color: #0000cd;
+    background-color: #0000cd70;
+} */
```
Comment (partha8) [inline packages/components/alert.css]: remove the comments

**Review by partha8 — APPROVED** [review]

**Pull request merged**